### PR TITLE
Add CSS3 text-emphasis

### PR DESCRIPTION
**Pull request by zmmbreeze**

### CODE REVIEW

**Comment by yisibl** [comment]
+1

**Comment by Fyrd** [comment]
Looks good, though I see you're using the `p` value for partial, which is actually for "polyfill". Needs to be replaced with `a` which is the character used for partial. Thanks!

**Comment by yisibl** [comment]
Please as soon as possible to merge.

**Comment by zmmbreeze** [comment]
@Fyrd I replaced `p` with `a`. :smile:

**Comment by ai** [comment]
@Fyrd next Autoprefixer releas requires this PR

File: features-json/text-emphasis.json [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+      "11":"n"
+    },
+    "and_uc":{
+      "9.9":"n"
+    }
+  },
+  "notes":"Some old webkit browser(like Chrome 24) support `-webkit-text-emphasis`, but `element.style.WebkitTextEmphasis` get an empty string, and not support CKJ language and `text-emphasis-position`. So it's basicly useless and marked as unsupported in this table.",
+  "notes_by_num":{
+    "1": "Some old webkit browser support `-webkit-text-emphasis-position`, but it dosn't follow the spec. It support `-webkit-text-emphaisi-position:under;`, but not support `-webkit-text-emphasis-position:under right;` which is the right one.",
+    "2": "Not support `text-emphasis-position:left;` or `text-emphasis-position:under left;`."
```
Comment (cvrebert) [inline features-json/text-emphasis.json]: "Does not support"

File: features-json/text-emphasis.json [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+      "10":"n",
+      "11":"n"
+    },
+    "and_uc":{
+      "9.9":"n"
+    }
+  },
+  "notes":"Some old webkit browser(like Chrome 24) support `-webkit-text-emphasis`, but `element.style.WebkitTextEmphasis` get an empty string, and not support CKJ language and `text-emphasis-position`. So it's basicly useless and marked as unsupported in this table.",
+  "notes_by_num":{
+    "1": "Some old webkit browser support `-webkit-text-emphasis-position`, but it dosn't follow the spec. It support `-webkit-text-emphaisi-position:under;`, but not support `-webkit-text-emphasis-position:under right;` which is the right one.",
```
Comment (cvrebert) [inline features-json/text-emphasis.json]: Bad grammar and typos:
> Some old webkit **browsers** support `-webkit-text-emphasis-position`, but **don't** follow the spec. **They** support -webkit-text-**emphasis**-position:under;, but **don't** support

File: features-json/text-emphasis.json [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    },
+    "ie_mob":{
+      "10":"n",
+      "11":"n"
+    },
+    "and_uc":{
+      "9.9":"n"
+    }
+  },
+  "notes":"Some old webkit browser(like Chrome 24) support `-webkit-text-emphasis`, but `element.style.WebkitTextEmphasis` get an empty string, and not support CKJ language and `text-emphasis-position`. So it's basicly useless and marked as unsupported in this table.",
```
Comment (cvrebert) [inline features-json/text-emphasis.json]: "basically"

**Comment by Fyrd** [comment]
I'll merge this and make @cvrebert's changes. Thanks all!

**Pull request merged**